starter-project-backend/src/controllers/article.ts

```diff
@@ -0,0 +1,142 @@
+import {Response, Request, NextFunction} from "express"
+import Article  from "../models/article"
+
+async function getAllArticle(req:Request, res:Response, next:NextFunction) 
+{
+  try{
+    let articles = await Article.find({}) 
+    res.status(200).json({data: articles})
```

Comment:
instead of using let you can use const since the object is not changing in this scope

Reply:
fixed